### PR TITLE
Cosmetics fixes of backtick in HTTP2 pages and added me as contributor.

### PR DESCRIPTION
**Pull request by lucab85**
Cosmetics fixes for better readability of back-tick in HTTP2 pages (en, es, id, ja, ko, pt-br, zh-cn) and add me as contributor.

### CODE REVIEW

File: src/content/en/fundamentals/performance/http2/index.md [hunk 1]
```diff
@@ -254,7 +254,7 @@ reassemble them on the other end.
 ![HTTP/2 request and response multiplexing within a shared connection](images/multiplexing01.svg)
 
 The snapshot captures multiple streams in flight within the same connection. The
-client is transmitting a DATA frame (stream 5) to the server, while the server
+client is transmitting a `DATA` frame (stream 5) to the server, while the server
```
Comment (jpmedley) [inline src/content/en/fundamentals/performance/http2/index.md]: Unless I'm misreading the HTTP2 spec, DATA as it's used here is nothing that will every appear in code. Meaning it's more like 'response header' than '`Response` instance'. (Notice the formatting.) Please remove the backticks.

File: src/content/en/fundamentals/performance/http2/index.md [hunk 2]
```diff
@@ -473,19 +473,19 @@ the same results, but with additional performance benefits. Push resources can b
 * Prioritized by the server
 * Declined by the client
 
-### PUSH_PROMISE 101
+### `PUSH_PROMISE` 101
```
Comment (jpmedley) [inline src/content/en/fundamentals/performance/http2/index.md]: Character formatting isn't used in headers.

File: src/content/en/fundamentals/performance/http2/index.md [hunk 3]
```diff
@@ -473,19 +473,19 @@ the same results, but with additional performance benefits. Push resources can b
...
 
-All server push streams are initiated via PUSH_PROMISE frames, which signal the
+All server push streams are initiated via `PUSH_PROMISE` frames, which signal the
```
Comment (jpmedley) [inline src/content/en/fundamentals/performance/http2/index.md]: This is the same issue as with DATA, above.

File: src/content/en/fundamentals/performance/http2/index.md [hunk 4]
```diff
@@ -511,13 +511,13 @@ resource and its properties. In HTTP/1.x, this metadata is always sent as plain
 text and adds anywhere from 500–800 bytes of overhead per transfer, and
 sometimes kilobytes more if HTTP cookies are being used. (See
 [Measuring and Controlling Protocol Overhead](https://hpbn.co/http1x/#measuring-and-controlling-protocol-overhead)
-.) To reduce this overhead and improve performance, HTTP/2 compresses request
+). To reduce this overhead and improve performance, HTTP/2 compresses request
```
Comment (jpmedley) [inline src/content/en/fundamentals/performance/http2/index.md]: It was correct before.

File: src/content/en/fundamentals/performance/http2/index.md [hunk 5]
```diff
@@ -565,15 +565,15 @@ and simple to implement correctly, and of course, enable good compression of
...
+[IETF HPACK - Header Compression for HTTP/2](https://tools.ietf.org/html/draft-ietf-httpbis-header-compression).
 
 ## Further reading:
 
 * [“HTTP/2”](https://hpbn.co/http2/){: .external }
     – The full article by Ilya Grigorik
 * [“Setting up HTTP/2”](https://surma.link/things/h2setup/){: .external }
     – How to set up HTTP/2 in different backends by Surma
-* [“HTTP/2 is here, let’s optimize!”](https://docs.google.com/presentation/d/1r7QXGYOLCh4fcUq0jDdDwKJWNqWK1o4xMtYpKZCJYjM/edit#slide=id.p19)
+* [“HTTP/2 is here, let’s optimize!”](https://docs.google.com/presentation/d/1r7QXGYOLCh4fcUq0jDdDwKJWNqWK1o4xMtYpKZCJYjM/edit#slide=id.p19)
```
Comment (jpmedley) [inline src/content/en/fundamentals/performance/http2/index.md]: I know this isn't your text, but as long as we're here, 'let's optimize' should actually be the start of a new sentence. Can you please take care of that?

File: src/content/en/fundamentals/performance/http2/index.md (hunk 1 above)
Comment (lucab85) [inline src/content/en/fundamentals/performance/http2/index.md]: IMHO the [RFC 7540 sec. 6](https://tools.ietf.org/html/rfc7540#section-6) specify `DATA` and `PUSH_PROMISE` frames meaning like markup code in HTML so I think the formatting is more correct with backticks. If I'm wrong I'll remove that from my commit.

File: src/content/en/fundamentals/performance/http2/index.md (hunk 2 above)
Comment (lucab85) [inline src/content/en/fundamentals/performance/http2/index.md]: I'll correct it.

File: src/content/en/fundamentals/performance/http2/index.md (hunk 1 above)
Comment (lucab85) [inline src/content/en/fundamentals/performance/http2/index.md]: I've re-read the paragraph and I was wrong. I'll corrent and commit again. Thank you for your feedback.

**Comment by jpmedley** [comment]
Thank you for your help. Ping me when you're done so I can merge.

Joe Medley | Technical Writer, Chrome DevRel | jmedley@google.com |
 816-678-7195
*If an API's not documented it doesn't exist.*

File: src/content/en/fundamentals/performance/http2/index.md (hunk 1 above)
Comment (lucab85) [inline src/content/en/fundamentals/performance/http2/index.md]: I've re-read the paragraph and I was wrong. I'll corrent and commit again. Thank you for your feedback.

**Pull request merged**